doc/source/text.rst

```diff
@@ -199,6 +199,143 @@ regular expression object will raise a ``ValueError``.
     ---------------------------------------------------------------------------
     ValueError: case and flags cannot be set when pat is a compiled regex
 
+.. _text.concatenate:
+
+Concatenation
+-------------
+
+There are several ways to concatenate a ``Series``, either with itself or others, all based on :meth:`~Series.str.cat`.
+
+Concatenating a single Series into a string
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The content of a ``Series`` can be concatenated:
+
+.. ipython:: python
+
+    s = pd.Series(['a', 'b', 'c', 'd'])
+    s.str.cat(sep=',')
+    
+If not specified, the keyword ``sep`` for the separator defaults to ``''``:
+
+.. ipython:: python
+
+    s.str.cat()
+
+By default, missing data is ignored. Using ``na_rep``, they can be given a representation:
+
+.. ipython:: python
+
+    t = pd.Series(['a', 'b', np.nan, 'd'])
+    t.str.cat(sep=',')
+    t.str.cat(sep=',', na_rep='-')
+
+Concatenating a Series and something one-dimensional into a Series
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The first argument to :meth:`~Series.str.cat` (which corresponds to the keyword ``others``) can be used to specify values that should be concatenated correspondingly. 
```

Comment:
can you make this a bit shorter / simpler. you don't need to explain everything, just a bit of text.

Reply:
its also not really relevant to say it can be a list, ndarray or Series, that is the purpose of a doc-string

Reply:
It was a first attempt. You said conversation style, and the way I wrote it, the features get introduced one-by-one.